client/src/components/Home/Home.ts

```diff
@@ -65,10 +65,10 @@ export default Vue.extend({
         }
     },
     mounted() {
-        axios
-            .get(API_URL)
-            .then(response => this.urls = response.data)
-            .catch(error => this.errorMessage = 'hmm... It seems network is broken or server is down 😴');
```

Comment:
First of all, thank you for your efforts. I can see you have removed a few lines. Any specific reason for that? I am curious how the application will behave if there isn't any network?

Reply:
oh right, my bad, I forgot to put back the comment. fixed now.

Reply:
Awesome.